### ISSUE TITLE
Remove Ppxlib's label exposed

### ISSUE THREAD

**Issue by davesnx** [ocaml-ppx/ppxlib#481]
There seems to be a type "label" assigned to string which messes a bit with error messages and inference when you use ppxlib with `open Ppxlib`. Is there any chance to fix it?

I endup doing a workaround like (unsure if there's a better way)

```
type s = string
open Ppxlib
type string = s
```

**Comment by NathanReb** [comment]
I agree this is annoying but it comes from the `Asttypes` that are in scope when opening `Ppxlib`. We obviously can't remove them from the scope as it would be a huge breaking change.

I'll try to see if we can incorporate a similar trick to yours directly into ppxlib.
